lib/commands/push.ts

```diff
@@ -224,6 +225,15 @@ export default class PushCmd extends Command {
 			char: 'g',
 			exclusive: ['multi-dockerignore'],
 		}),
+		tag: flags.string({
+			description: stripIndent`
+				When performing a push to an application, use --tag in order set the
+				release version of the produced release with a string of your choice.
+				e.g:
+					--tag v1.0
```

Comment:
I think that this example could be added to the examples section of the documentation (`public static examples` array), rather than here. Also, this description mentions "release version", which we have learned is a different feature for another PR. Suggestion:
```
tag: flags.string({
	description: stripIndent`
		Create a new release tag with the given key[:value] pair. The value is optional.
		May be specified multiple multiple times, in which case it must be specified
		at the end of the command line.`
```
See other comment re `multiple: true`. The part regarding _"in which case it must be specified at the end of the command line"_ needs testing / confirmation. (Ideally it would not be needed.)